reloj_checador_team03/app/helpers/attendances_helper.rb

```diff
@@ -0,0 +1,2 @@
+module AttendancesHelper
```

Comment:
If you don't need helpers, delete it. This was recommended to me

Reply:
thanks